lib/paypalhttp/serializers/form_encoded.rb

```diff
@@ -1,11 +1,11 @@
-require 'uri'
+require 'cgi'
 
 module PayPalHttp
   class FormEncoded
     def encode(request)
       encoded_params = []
       request.body.each do |k, v|
-        encoded_params.push("#{URI.escape(k.to_s)}=#{URI.escape(v.to_s)}")
+        encoded_params.push("#{CGI.escape(k.to_s)}=#{CGI.escape(v.to_s)}")
```

Comment:
Just curious, I see multiple PRs with this change with various implementations:

https://github.com/paypal/paypalhttp_ruby/pull/18
https://github.com/paypal/paypalhttp_ruby/pull/13
https://github.com/paypal/paypalhttp_ruby/pull/4

Is there a reason you chose this route over the others?

Reply:
The reason we decided to choose this implementation over the others is because it is more consistent with other maintained Paypal related repos. One repo in particular that we looked at was the [braintree_ruby repo](https://github.com/braintree/braintree_ruby). There are multiple instances where `CGI.escsape` is utilized, so we believed it was best to continue this consistency of using this specific function with repo as well.

Reply:
:+1: Thanks for the explanation